src/app/store/subnet/types/base.ts

```diff
@@ -120,3 +125,14 @@ export type SubnetState = GenericState<Subnet, APIError> & {
   eventErrors: EventError<Subnet, APIError, SubnetMeta.PK>[];
   statuses: SubnetStatuses;
 };
+
+export type StaticDHCPLease = {
+  id: number;
+  comment: string | null;
+  ip_address: string;
+  mac_address: string;
+  interface: string;
+  node: SimpleNode;
```

Comment:
I had some reservations about passing through a node, but given we'll need the ID as well to make the links work, I think this should be fine for now.